src/services/item-published/types.ts

```diff
@@ -6,5 +6,5 @@ export interface ItemPublished {
   creator?: Member;
   createdAt: Date;
   item: DiscriminatedItem;
-  totalViews: number;
+  totalViews?: number;
```

Comment:
Why has this become optional ?

Reply:
as ItemPublished type used for DB table which doesn't have a totalViews column

Reply:
@LinaYahya Ah yes we changed that in https://github.com/graasp/graasp/pull/580 now DB table types do not implement sdk types since sdk types represent the API layer. DB types are their own thing. sdk types can be used in services given you supply a mapping from entity type to api type (which we should do).

Reply:
Yes this should solve the problem, I'll revert the change